src/FSAL/FSAL_CORTXFS/mds.c

```diff
@@ -378,7 +510,8 @@ int kvsfs_pmds_fini(struct kvsfs_fsal_module *kvsfs)
 	return rc;
 }
 
-
+#if 0
```

Comment:
If this block is not needed, please clean it.

Reply:
The reason i did not remove this function is because code from this function can be leveraged when we dynamically update the DS list.